storage/backend/ceph/ceph.go

```diff
@@ -150,7 +150,7 @@ func (c *Driver) Format(do storage.DriverOptions) error {
 
 // Destroy a volume.
 func (c *Driver) Destroy(do storage.DriverOptions) error {
-	poolName := do.Volume.Params["pool"]
+	poolName := do.Volume.Params["pool"].(string)
```

Comment:
https://github.com/yuva29/volplugin/blob/036c561893e8b778b038eb92e78abda1b83b9d01/storage/backend/ceph/ceph.go#L557-L564 this validation ensures that its string or any expected type and this validation is called prior to any storage operations. Do we still need to capture the conversion error?